ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -30,7 +30,10 @@
 import tempfile
 import time
 from pipes import quote
-import configparser
+try:
+    import configparser
+except ImportError:
+    import ConfigParser
```

Comment:
``import ConfigParser as configparser``

Reply:
or from configparser/ConfigParser import RawConfigParser

then change configparser.RawConfigParser to just RawConfigParser

Reply:
That can be simplified with:
from six.moves import configparser

Reply:
fixed